components/Palettes/src/UIColor+MDCPalettable.h

```diff
@@ -0,0 +1,28 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+#import "MDCPalettes.h"
+
+@protocol MDCPalettable <NSObject>
```

Comment:
I'm surprised to see us defining a protocol in a class extension.

Reply:
I agree and this can easily be removed - this was a quick-and-dirty proof of concept, so I didn't do a lot of clean up along the way.